css/styles.css

```diff
@@ -0,0 +1,526 @@
+}
+
+.title_phrase2 {
+  width: 14%;
+  margin-top: -15px;
+}
+
+.thirdArticle {
+  display: flex;
+  flex-direction: row;
+  width: 49%;
+  height: 250px;
+  background-color: #666;
+}
+
+.thirdArticlesSection {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 1% 3%;
+}
+
+.thirdArticle_thirdArticleText {
+  margin-left: 5%;
+}
+
+.thirdArticleTextP {
+  font-size: 1rem;
+  line-height: 1.5;
+}
+
+.thirdArticleTitleP {
+  font-size: 1.5rem;
+  line-height: 5rem;
+}
+
+.thumbAuthor {
+  height: 4rem;
+  cursor: pointer;
+  transform: rotateZ(calc(-11 * 1deg));
+  transition: transform 0.2s ease-out;
+  border: 5px solid white;
+  border-radius: 10px;
+}
+
+.thumbAuthor:hover {
+  transform: rotateZ(0);
+  transition: transform 0.2s ease-out;
+}
+
+.topArticlesSection {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.topMenuA {
+  padding: 15px;
+  border-radius: 12px;
+}
+
+.topMenuA:hover {
+  background-color: #b43125;
+}
+
+.topMenuA_moreButton {
+  background-color: #888;
+}
+
+.topMenuA_moreButton:hover {
+  background-color: #8e3a30;
+}
+
+.topSearchInput {
+  width: 350px;
+  height: 56px;
+  border-radius: 10px;
+  border: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1em;
+  font-weight: 700;
+  line-height: 1.5em;
+  padding: 5px 15px;
+  box-sizing: border-box;
+}
+
+.fourthArticle {
+  display: flex;
+  justify-content: space-around;
+}
+
+.fourthArticlesection_fourthArticle {
+  background-color: white;
+  padding: 2rem;
+}
+
+.fourthArticleHeading {
+  color: black;
+  padding: 20px;
+  font-weight: lighter;
+  font-size: small;
+}
+
+.fourthArticleText {
+  font-size: xx-large;
+  color: #444;
+  padding: 20px;
+}
+
+.fourthArticleContent {
+  color: black;
+  padding: 20px;
+}
+
+.fourthArticlebutton {
+  color: white;
+  background-color: #444;
+  padding: 20px;
+  margin: 20px;
+  border-radius: 5px;
+  cursor: pointer;
+  font-size: larger;
+}
+
+.footerheading {
+  color: white;
+  text-align: center;
+  margin-top: 5rem;
+}
+
+.footerfirstpart {
+  color: white;
+  display: flex;
+  margin-left: 8rem;
+  margin-top: 5rem;
+}
+
+.footerfirstpartitems {
+  padding-right: 38px;
+}
+
+.footerfirstpartitems :hover {
+  color: black;
+}
+
+.footersecondpart {
+  color: white;
+  display: flex;
+  margin-left: 8rem;
+  margin-top: 1rem;
+  padding-left: 3rem;
+}
+
+.footersecondpartitems {
+  padding-right: 38px;
+}
+
+.footersecondpartitems :hover {
+  color: black;
+}
+
+.footerthirdpart {
+  color: white;
+  display: flex;
+  margin-left: 8rem;
+  margin-top: 1rem;
+  padding-left: 1rem;
+}
+
+.footerthirdpartitems {
+  padding-right: 38px;
+}
+
+.footerthirdpartitems :hover {
+  color: black;
+}
+
+.footerfourthpart {
+  color: white;
+  display: flex;
+  margin-left: 8rem;
+  margin-top: 1rem;
+  padding-left: 25rem;
+}
+
+.footerfourthpartitems {
+  padding-right: 38px;
+}
+
+.footerfourthpartitems :hover {
+  color: black;
+}
```

Comment:
This selector has the same style definition of the selectors on line 406, 422, and 438  group selectors with the same style definition.